### ISSUE TITLE
Data Migration Bug in 4.7

### ISSUE THREAD

**Issue by streamzyuk** [nopSolutions/nopCommerce#6972]
nopCommerce version: 4.7 currenct source code

Steps to reproduce the problem: Upgrade to current dev master

When upgrading data in src/Libraries/Nop.Data/Migrations/UpgradeTo470
/DataMigration.cs

Deleting IsTelecommunicationsOrBroadcastingOrElectronicServices column from products table causes migration to fail. 

because IsTelecommunicationsOrBroadcastingOrElectronicServices column name is longer than 53 chars.

Should be renamed first to a shorter name to be able to be deleted.

Apart from that I am shocked with the performance. App starts in 5 seconds and the the pages are crazy fast even in debug mode and even on first call. I don't know if its because of Net 8 or code improvements.

**Comment by skoshelev** [comment]
Hi @streamzyuk. Please tell us what DBMS you are using and what version, and if possible, provide the entire error stack so that we can better understand the problem. As far as we know, the MsSql server supports up to 128 characters in column names by default.

**Comment by streamzyuk** [comment]
Hi I use Mysql 8.0

I can't share the full stack error as I cleared the logs.

But I tried to reproduce the case. Created the column IsTelecommunicationsOrBroadcastingOrElectronicServices  manually on product table and deleted the migration version record and rerun the migration. This time it deleted the column without errors.

I don't know why I got the exception at first place. But it was complaining the about the length of the column.
